apps/arweave/src/ar_data_sync.erl

```diff
@@ -1600,6 +1685,13 @@ read_chunk_with_metadata(
 					case ar_sync_record:is_recorded(Offset, StoredPacking, ?MODULE,
 							StoreID) of
 						false ->
+							?LOG_ERROR([{event,
```

Comment:
Similar comment as above, what do you tink about logging `ets:lookup(sync_records, {ID, Type, StoreID})` to see if maybe the `sync_records` db has gotten corrupt somehow so even the top level key is not found.